### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by TomasRoj**
This PR includes first three examples for beginners to learn basics pinout and work with shield. These examples are very interesting but some (that third) isnt very helpful. Third example is only for fun.

### CODE REVIEW

**Comment by TomasRoj** [comment]
IDK why. But travis test doesnt work. Even if the code is ok.

File: examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+Série příkladů, které vás seznámí s funkcemi tohoto shieldu.
+Vytvořeno @RoboticsBrno
+*/
+
+#import <ALKS.h>
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino:6]: Why are you using `#import`?

File: examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*
+Série příkladů, které vás seznámí s funkcemi tohoto shieldu.
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino:2]: Use English only, applies to comments, variable names etc, you should also respect headers of existing examples (use them).

**Comment by JarekParal** [comment]
Sorry, but all of your examples are wrong:
- you don't use the ALKS library for [pinout](https://github.com/RoboticsBrno/ArduinoLearningKitStarter-library/blob/master/README.md) - why do you think that we have this library
- you don't respect the examples [folder structure](https://github.com/RoboticsBrno/ArduinoLearningKitStarter-library/tree/master/examples) - has some folder name `ALKS` in name 
- you don't have English description of the source code, as our examples - we want all the documentation in English
- you don't respect compiler warning as [Travis report](https://travis-ci.org/RoboticsBrno/ArduinoLearningKitStarter-library/builds/520399584#L544)
- you created 3 same examples with `buzzer_song...` - one is enought but still, we haven't got buzzer - we have `piezo` in documentation
- [Travis.ci faild](https://travis-ci.org/RoboticsBrno/ArduinoLearningKitStarter-library/builds/520399584) because you have same `.ino` files in one folder

Sorry, but we don't want such examples.

**Review by cleveHEX — CHANGES_REQUESTED** [review]

**Comment by JarekParal** [comment]
PS: You probably copy the source code of `buzzer_song` from some other repository. Then there must be a link to the source of the code.

**Comment by TomasRoj** [comment]
Oh fuck.

**Comment by TomasRoj** [comment]
Okay then I will close it

**Comment by JarekParal** [comment]
@TomasRoj  you can improve your examples. But you should respect our requirements on the code quality.

**Comment by TomasRoj** [comment]
@JarekParal let me just little bit explain it. I am teen and I do little bit of coding. So I don't know much about github and you. So I don't know much about these things but I want learn so yes I will improve it but you know I need some help

**Comment by TomasRoj** [comment]
And what about that Readme pull request?

File: examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino:1]: One example folder can contain only one `*.ino`, has to have same name.

File: examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+*/
+
+#import <ALKS.h>
+
+//Importujeme potřebné tóny pro bzučák.
+#include "pitches.h" 
+#include "themes.h" 
+
+//Pin pro bzučák
+int bzucak = 9;
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino:13]: Invalid, import `ALKS.h` and use its pin name.

File: examples/ALKS_BuzzerSongs/buzzerSongs_Pirates.ino [hunk 5]
```diff
@@ -0,0 +1,31 @@
+/*
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/buzzerSongs_Pirates.ino:1]: Second `.ino` in folder

File: examples/ALKS_BuzzerSongs/buzzerSongs_Titanic.ino [hunk 6]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/buzzerSongs_Titanic.ino:1]: Third`.ino` in folder

File: examples/ALKS_BuzzerSongs/pitches.h [hunk 7]
```diff
@@ -0,0 +1,90 @@
+
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/pitches.h:1]: This seems like it is not your work, you signed a CLA, where you stated everything is written by you, or you gave appropriate credit (where license allows).

File: examples/ALKS_BuzzerSongs/themes.h [hunk 8]
```diff
@@ -0,0 +1,91 @@
+/*
```
Comment (cleveHEX) [inline examples/ALKS_BuzzerSongs/themes.h:1]: As above.

File: examples/ALKS_SmartLed/blink.ino [hunk 9]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (cleveHEX) [inline examples/ALKS_SmartLed/blink.ino:1]: Invalid name of `.ino` file again. I don't like this example at all.

File: examples/ALKS_pwmLed/pwmLed.ino [hunk 10]
```diff
@@ -0,0 +1,29 @@
+#import <ALKS.h>
```
Comment (cleveHEX) [inline examples/ALKS_pwmLed/pwmLed.ino:1]: Same as previous comments.

**Review by cleveHEX — CHANGES_REQUESTED** [review]
You could start here.

File: examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino (hunk 3 above)
Comment (TomasRoj) [reply examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino:1]: Can i have pitches.h and themes.h in that folder?

Comment (cleveHEX) [reply examples/ALKS_BuzzerSongs/buzzerSongs_CrazyFrog.ino:1]: Yes, you can.

**Pull request closed without merging**